libs/core/popover/popover-service/popover.service.ts

```diff
@@ -230,12 +231,26 @@ export class PopoverService extends BasePopoverClass {
         }
     }
 
+    /** Changes background theming when modal */
+    checkModalBackground(): void {
+        const bodyClass = 'fd-overlay-active';
+        const triggerClass = 'fd-popover__modal';
+        if ((!this.closeOnOutsideClick || !this.closeOnEscapeKey) && this.isOpen) {
+            this._renderer.addClass(document.body, bodyClass);
+            this._renderer.addClass((this._triggerElement as ElementRef).nativeElement, triggerClass);
+        } else if ((!this.closeOnOutsideClick || !this.closeOnEscapeKey) && !this.isOpen) {
```

Comment:
We also may want to see this removal logic as part of the destroy process for the service